kedro/framework/cli/project.py

```diff
@@ -90,7 +98,13 @@ def test(metadata: ProjectMetadata, args, **kwargs):  # pylint: disable=unused-a
 def lint(
     metadata: ProjectMetadata, files, check_only, **kwargs
 ):  # pylint: disable=unused-argument
-    """Run flake8, isort and black."""
+    """Run flake8, isort and black. (DEPRECATED)"""
+    deprecation_message = (
+        "DeprecationWarning: Command 'kedro lint' is deprecated and "
+        "will not be available from Kedro 0.19.0."
```

Comment:
Do we need to point out that users should call linters directly? Do we need to specify which ones?

Reply:
I am less sure about this one, maybe it will become more clear when #1620 is done. I am fine we either wait until #1620 is done or just merge this one and add some links when the docs PRs are done.